theme/documentation/article_pages.md

```diff
@@ -25,6 +25,10 @@ This article is to be used to promote knowledge sharing and to provide solutions
 
 This article is built on top of the **Knowledge Base Articles** designed to be part of Liferay's fast-track publication program of providing a repository of verified solutions that have resolved one or more customer issues. It contains a fast track label at the top of the article, as well as a brief explanation about what fast track articles are at the bottom of the article. Articles published under this template are part of the Knowledge-Centered Service initiative.
 
+### System Status article
+
+This is a one-off article template based on the Zendesk default article to display a system outage warning at the top of the page for DXP cloud.
```

Comment:
We can update this to: `This is a one-off article template used to display a system outage warning at the top of the page for DXP cloud.  This is based on Zendesk's default article.`

Reply:
ccb4116